web/src/components/dashboard/AnytimeActionDropdown.test.tsx

```diff
@@ -323,5 +323,79 @@ describe("<AnytimeActionDropdown />", () => {
 
       expect(screen.queryByText("license - hvac-reinstatement")).not.toBeInTheDocument();
     });
+
+    it("adds vacant property anytime action for vacant property owners", () => {
+      anytimeActionTasks = [
+        generateAnytimeActionTask({ filename: "vacant-building-fire-permit" }),
+        ...anytimeActionTasks,
+      ];
+      useMockBusiness({
+        profileData: generateProfileData({
+          vacantPropertyOwner: false,
+        }),
+      });
+      renderAnytimeActionDropdown();
+      fireEvent.click(screen.getByLabelText("Open"));
+      expect(screen.queryByTestId("vacant-building-fire-permit-option")).not.toBeInTheDocument();
+
+      useMockBusiness({
+        profileData: generateProfileData({
+          vacantPropertyOwner: true,
+        }),
+      });
+      renderAnytimeActionDropdown();
+      fireEvent.click(screen.getByLabelText("Open"));
+      expect(screen.getByTestId("vacant-building-fire-permit-option")).toBeInTheDocument();
+    });
+
+    it("adds fire carnival modification for carnival ride owning businesses", () => {
+      anytimeActionTasks = [
+        generateAnytimeActionTask({ filename: "carnival-ride-supplemental-modification" }),
+        ...anytimeActionTasks,
+      ];
+      useMockBusiness({
+        profileData: generateProfileData({
+          carnivalRideOwningBusiness: false,
+        }),
+      });
+      renderAnytimeActionDropdown();
+      fireEvent.click(screen.getByLabelText("Open"));
+      expect(screen.queryByTestId("carnival-ride-supplemental-modification-option")).not.toBeInTheDocument();
+
+      useMockBusiness({
+        profileData: generateProfileData({
+          carnivalRideOwningBusiness: true,
+        }),
+      });
+      renderAnytimeActionDropdown();
+      fireEvent.click(screen.getByLabelText("Open"));
+      expect(screen.getByTestId("carnival-ride-supplemental-modification-option")).toBeInTheDocument();
+    });
+
+    it("adds operating carnival fire permit for carnvial owning businesses and traveling circuses owning businesses", () => {
```

Comment:
The case returns a boolean, there is a return statement that has an OR, not and AND. I recommend creating 2 test, each containing one part of the conditional that is false.

      case "operating-carnival-fire-permit":
        return (
          !!business?.profileData.carnivalRideOwningBusiness ||
          !!business?.profileData.travelingCircusOrCarnivalOwningBusiness
        );